examples/apache2-vhost.conf

```diff
@@ -0,0 +1,40 @@
+WSGIScriptAlias / /opt/integreat-cms/src/backend/wsgi.py
```

Comment:
I'm not sure if "examples" is a good directory name here, maybe rather "apache-conf"/"prod-conf"/"example-conf" or similar?

Reply:
My idea here was to potentially collect other example config files here as well. What about `example-configs` ?

Reply:
ok, sounds good :+1: